test/unit/parse-test.coffee

```diff
@@ -40,7 +39,11 @@ describe('Parsing API description document', ->
         assert.isTrue(apiElements.annotations?.isEmpty)
       )
       it('the parse result contains source map elements', ->
-        assert.include(JSON.stringify(serialize(apiElements)), '"sourceMap"')
+        sourceMaps = apiElements
+          .recursiveChildren
+          .map((element) -> element.sourceMapValue)
+          .filter((sourceMap) -> !!sourceMap)
```

Comment:
Using https://github.com/refractproject/minim/pull/149 this could become:

```js
sourceMaps = apiElements.recursiveChildren.flatMap((element) -> element.sourceMapValue)
```